### ISSUE TITLE
get_track_audio_features that takes artist name & title

### ISSUE THREAD

**Issue by JosiahParry** [charlie86/spotifyr#76]
Hey Charlie,

I came across a scenario this week where I wanted to be able to query a single track's audio features based on artist name and track title. I implemented a local function to do this. I'm wondering if this is a feature that could be added to spotifyr. Perhaps this use case is too narrow.

The specific example was querying billboard top charts tracks and retrieving audio features and lyrics based solely on the track title and artist name.

https://gist.github.com/JosiahParry/f5e8384dd9e8f8903640bb7ae319807c

**Comment by charlie86** [comment]
@JosiahParry this is a great idea - I've been meaning to add this into the existing `get_track_audio_features` function, so that it could take either a list of ids or artist/track name

**Comment by JosiahParry** [comment]
I'd definitely support this. It could lower the barrier to entry of the API a bit. Navigating all of the different IDs can be a bit troublesome I think

**Comment by drew-walkerr** [comment]
Is it possible to use this for a list of track names/artists?

**Comment by JosiahParry** [comment]
@drew-walkerr absolutely. See the example in [mirr](https://mirr.netlify.com/audio-features-spotify.html)
